crates/cargo-test-macro/src/lib.rs

```diff
@@ -59,6 +61,56 @@ pub fn cargo_test(attr: TokenStream, item: TokenStream) -> TokenStream {
     ret
 }
 
+#[proc_macro]
+pub fn find_tests(input: TokenStream) -> TokenStream {
+    let mut ret = TokenStream::new();
+    // remove quotations from the input string
+    let input_dir = input.to_string().replace("\"", "");
```

Comment:
I think it'd be worth hardening how we process the parameter list for this macro

Reply:
I'm not quite sure how to do that